### PR TITLE
New front end index and login done

### PR DESCRIPTION
**Pull request by carlguo866**

### CODE REVIEW

File: jiezi/learning/views.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 def index(request):
     # because the index page isn't ready
-    return redirect('/learning/C0001')
+    return render(request,"index.html")
```
Comment (chenyx512) [inline jiezi/learning/views.py]: use single quotation mark

File: jiezi/templates/base.html [hunk 2]
```diff
@@ -4,49 +4,51 @@
...
+    <div class='main-content-container' id='footer'>
+        <p>FOOTER NEEDS TO BE IMPLEMENTED</p>
+    </div>
 </footer>
 </body>
-
 <script src="{% static 'scripts/main.js' %}"></script>
-{% block js %}{% endblock js %}
+{% block extra_script %}{% endblock %}
 </html>
```
Comment (chenyx512) [inline jiezi/templates/base.html:56]: We have already discussed that we are using an indention of 2 for html

File: jiezi/templates/index.html [hunk 3]
```diff
@@ -1,82 +1,95 @@
...
-  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-	<meta name='viewport' content='width=device-width, initial-scale=1'>
+{% extends 'base.html' %}
+{% load static %}
+{% block css %}
+    <link rel='stylesheet' type="text/css" href='{% static "styles/dashboard.css" %}'>
+{% endblock %}
+{% block content %}
+    <h1>Welcome back, {{ user.first_name }}!</h1>
+    <a href="{% url "manage_library" %}"class='button button-primary' href='/vocab.html'>Manage Vocab List</a>
```
Comment (chenyx512) [inline jiezi/templates/index.html]: here use single quotation mark for outside and double for inside, or the other way

File: jiezi/templates/registration/login.html [hunk 4]
```diff
@@ -1,45 +1,46 @@
...
+            <h1>Login</h1>
+            <div class='input-wrapper username'>
+                <input name="username" class='input-box' type='text' placeholder='Username'></input>
+            </div>
+            <div class='input-wrapper password'>
+                <input class='input-box' name= 'password' type='password' placeholder='Password'></input>
+            </div>
+            <div id='login-button-wrapper'>
+                <button class='button button-primary' value="Login" id='login-button' type="submit">Login</button>
+                  <input type="hidden" name="next" value="{{ next }}" />
```
Comment (chenyx512) [inline jiezi/templates/registration/login.html]: indention is incorrect here

**Review by chenyx512 — CHANGES_REQUESTED** [review]
Generally good, but these are a few things
1. The indention for html as we discuss should be 2 instead of 4
2. Stick to single quote
3. Do we actually have to have a slider?

File: jiezi/templates/index.html (hunk 3 above)
Comment (carlguo866) [reply jiezi/templates/index.html]: this is yjm's code style, but I can fix that

**Pull request merged**